### PR TITLE
don't blow up on stat errors

### PR DESCRIPTION
**Pull request by lordnull**
Status 'too_busy' returns default value. This means A warning is only
generated if a truely unexpected value is found.

This is a squash and minor change to the 1.3.3 patch.

### CODE REVIEW

File: src/riak_repl_console.erl [hunk 1]
```diff
@@ -952,14 +952,20 @@ proplists_get([], undefined, Default) ->
...
     Value;
+proplists_get([Key], Props, Default) when is_list(Props) ->
+    Value = proplists:get_value(Key, Props),
+    proplists_get([], Value, Default);
 proplists_get([Key | Path], Props, Default) when is_list(Props) ->
     case proplists:get_value(Key, Props) of
         undefined ->
             Default;
+        too_busy ->
+            Default;
```
Comment (bookshelfdave) [inline src/riak_repl_console.erl:964]: Should a lager:debug() be used here in case we want to see what's truly going on?

**Comment by bookshelfdave** [comment]
Should this PR be against 1.4 instead of master?

**Comment by lordnull** [comment]
We've got caa8cf4160a6c60d9757c2d5ed4cd424f29be70a in 1.4. This is not a big enough change to merit back porting. No data loss, no new crashes, nor any different handling of existing crashes (or in this case recently fixed crashes).

With that said, the branch name is wonderfully misleading.

**Comment by bookshelfdave** [comment]
+1

**Pull request merged**